partyTracker.lua

```diff
@@ -30,10 +35,11 @@ local function updateParty()
     
     -- Ensure existence of partyStack entity
     for i=1, GetNumGroupMembers() do
-        local fullName, realm, unitExists = util:GetNameRealm(prefix .. i)
-        -- print("fullName: " .. fullName)
-        if fullName ~= "Unknown" and fullName ~= prefix .. i and unitExists then -- If the user identifier cannot be fetched then we just omit (usually it happens when its us in the party)
-            local entryKey = util:GetCurrentRegion() .. "/" .. util:GetRealmSlug(realm) .. "/" .. fullName
+        local unitName = prefix .. i
+        local characterName, characterRealm = util:GetNameRealm(unitName)
+
+        if characterName and characterName ~= UNKNOWNOBJECT and characterName ~= unitName and characterRealm then -- If the user identifier cannot be fetched then we just omit (usually it happens when its us in the party)
```

Comment:
`unitExists` would've been the same as `prefix .. i`, thus always returned a truth-y value, causing the `if` to trigger even when it shouldn't. This argument has been removed from `GetNameRealm` as it was only used here. Now checks if the realm actually exists as without we can't do anything